### DISCUSSION TITLE
Wrong rect coordinate from getCurrentSelection()

### DISCUSSION THREAD

**Discussion by marshal1919** [readium/kotlin-toolkit#394] [Q&A]
When running test-app,some highlights in table content page would be misaligned.
While tarcking the running process I found that the rect coordinate returned by js:getCurrentSelection() was incorrect.
Why?
```javascript
export function getCurrentSelection() {
  const text = getCurrentSelectionText();
  if (!text) {
    return null;
  }
  const rect = getSelectionRect();
  return { text, rect };
}
``` 
```kotlin
private fun showHighlightPopupWithStyle(style: Highlight.Style) =
        viewLifecycleOwner.lifecycleScope.launch {
            viewLifecycleOwner.repeatOnLifecycle(Lifecycle.State.STARTED) {
                // Get the rect of the current selection to know where to position the highlight
                // popup.
                (navigator as? SelectableNavigator)?.currentSelection()?.rect?.let { selectionRect ->
                    showHighlightPopup(selectionRect, style)
                }
            }
        }
``` 

![Screenshot_2023-09-22-22-28-04-144_org readium r2reader](https://github.com/readium/kotlin-toolkit/assets/55966326/ecba22e9-9cc6-4bc0-861a-2063a77a95d2)

**Comment by mickael-menu** [comment]
Please open a bug report issue describing all the steps to reproduce the issue:

https://github.com/readium/kotlin-toolkit/issues/new?assignees=&labels=&projects=&template=bug_report.md&title=
